connectors/sources/confluence.py

```diff
@@ -714,6 +726,7 @@ async def fetch_server_space_permission(self, space_key):
             return {}
 
         url = URLS[SPACE_PERMISSION].format(space_key=space_key)
+        self._logger.debug(f"Fetching permissions for '{space_key}' space")
```

Comment:
Should this be `info` log-level as it indicates progress?

Reply:
Yes, this should be info.